lib/events/dynamoevents/dynamoevents.go

```diff
@@ -275,6 +317,92 @@ const (
 	tableStatusOK
 )
 
+// migrateRFD24WithRetry repeatedly attempts to kick off RFD 24 migration in the event
+// of an error on a long and jittered interval.
+func (l *Log) migrateRFD24WithRetry(ctx context.Context) {
+	for {
+		if err := l.migrateRFD24(ctx); err != nil {
+			delay := utils.HalfJitter(time.Minute * 5)
+			log.WithError(err).Errorf("Failed RFD 24 migration, making another attempt in %f seconds", delay.Seconds())
```

Comment:
`delay` is never actually used to wait

also, the delay interval seems rather long.
do we want to block the auth server startup for up to 5min because the first attempt at index creation failed?

Reply:
We could reduce this to something shorter like 1 minute without too much extra work being done I think, may run into some API errors from Dynamo more times on duplicate attempts at index modification but that's safe as long as we retry.

Reply:
Reduced to 1 minute in a recent commit.